src/confluent_kafka/admin/__init__.py

```diff
@@ -291,22 +297,23 @@ def _make_futures(futmap_keys, class_check, make_result_fn):
 
     def create_topics(self, new_topics, **kwargs):
         """
-        Create new topics in cluster.
-
-        The future result() value is None.
+        Create one or more new topics.
 
-        :param list(NewTopic) new_topics: New topics to be created.
-        :param float operation_timeout: Set broker's operation timeout in seconds,
+        :param list(NewTopic) new_topics: A list of specifictions (NewTopic) for
+                  the topics that should be created.
+        :param float operation_timeout: The operation timeout in seconds,
                   controlling how long the CreateTopics request will block
                   on the broker waiting for the topic creation to propagate
                   in the cluster. A value of 0 returns immediately. Default: 0
-        :param float request_timeout: Set the overall request timeout in seconds,
+        :param float request_timeout: The overall request timeout in seconds,
                   including broker lookup, request transmission, operation time
                   on broker, and response. Default: `socket.timeout.ms*1000.0`
-        :param bool validate_only: Tell broker to only validate the request,
+        :param bool validate_only: If true, the request is validated only,
```

Comment:
This reads weird now. "the request is only validated without creating the topic"